app/main/views/marketplace.py

```diff
@@ -75,14 +75,25 @@ def terms_and_conditions():
 def get_brief_by_id(framework_framework, brief_id):
     briefs = data_api_client.get_brief(brief_id)
     brief = briefs.get('briefs')
+    brief_responses = data_api_client.find_brief_responses(
+        brief_id=brief_id,
+        status='draft,submitted'
+    ).get("briefResponses")
+    started_brief_responses_count = len([response for response in brief_responses if "submittedAt" not in response])
+    completed_brief_responses_count = len([response for response in brief_responses if "submittedAt" in response])
+
     if brief['status'] not in ['live', 'closed', 'withdrawn'] or brief['frameworkFramework'] != framework_framework:
         abort(404, "Opportunity '{}' can not be found".format(brief_id))
 
-    if getattr(current_user, "supplier_id", None) is None:
-        # user unauthenticated or not a supplier
-        brief_responses = None
-    else:
-        brief_responses = data_api_client.find_brief_responses(brief_id, current_user.supplier_id)["briefResponses"]
+    has_supplier_responded_to_brief = False
+    if brief_responses and getattr(
```

Comment:
Because you're only using the boolean in the if statement this can be a `hasattr` call rather than a `getattr` call.

You could also use a try, except block to follow the beg forgiveness not permission paradigm like so:
```

try:
    has_supplier_responded_to_brief = current_user.supplier_id in [i['supplierId'] for i in brief_responses]
except AttributeError:
    has_supplier_responded_to_brief = False